server/database/schema.sql

```diff
@@ -796,6 +796,21 @@ CREATE TABLE IF NOT EXISTS toolset_prompts (
 CREATE UNIQUE INDEX IF NOT EXISTS toolset_prompts_toolset_id_prompt_name_key
 ON toolset_prompts (toolset_id, prompt_name);
 
+CREATE TABLE IF NOT EXISTS mcp_install_page_metadata (
```

Comment:
let's add project_id here. Where we have things like toolset it's generally a good idea to have project_id just to avoid accidental leakage